interface/main/finder/select_patient.php

```diff
@@ -0,0 +1,179 @@
+<?php
+// Copyright (C) 2012 Rod Roark <rod@sunsetsystems.com>
```

Comment:
Dunno why this file is showing up as new in this branch.

Reply:
Used Patient Finder as Select Patient in this PR with only difference being that former leads to demographics tab of a patient and latter leads to track appointments tab of a patient. Is there a need to change the header info.?